package.json

```diff
@@ -23,7 +23,7 @@
     "plaiceholder": "^2.5.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-loading-skeleton": "^3.1.0",
+    "react-loading-skeleton": "^3.1.1",
```

Comment:
delete the package-lock.json and run `yarn install`

Reply:
If you're following a tutorial that says `npm install` replace that with `yarn add` (same `npm install -D` → `yarn add --dev`)